bin/brew

```diff
@@ -45,5 +85,77 @@ fi
 
 export HOMEBREW_BREW_FILE
 export HOMEBREW_RUBY_PATH
+export HOMEBREW_PREFIX
+export HOMEBREW_REPOSITORY
+export HOMEBREW_LIBRARY
+export HOMEBREW_CELLAR
+
+if [ -n "$HOMEBREW_OSX" ]
+then
+  if [ "$(xcode-\select --print-path)" = "/" ]
+  then
+    cat >&2 <<EOS
```

Comment:
We need to check `stderr` with `[[ -t 2 ]]` as that's the stream we are dumping this to. Otherwise, I'd be fine with this implementation.

Rather cosmetic, but I think I would tend to replace `echo -n[e]` with `printf` (and include the colon into the highlighting) and I would use `[[ $# -eq 0 ]]` instead of `[[ $# == 0 ]]` (to get a numerical comparison; this is more about communicating intent than about the actual outcome).